modules/core/site.css

```diff
@@ -329,4 +329,105 @@ div.unseen, .unseen .subject { font-weight: 700; }
 }
 .drag_target {
     background-color: #888 !important;
+}
+
+.modal {
+    display: block;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+}
+
+.modal-bg {
+    background-color: rgba(0,0,0,0.4);
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+}
+  
+.modal-content {
```

Comment:
@jacob-js this is a good start for refactoring and making uniform cypht modals but as you can view in this comment https://github.com/cypht-org/cypht/pull/854#discussion_r1441515544, using class names used by popular frameworks can lead to unexpected behaviors when Cypht is embedded. Can you prefix all these classes with something like **cypht-**